ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_telemetry_postcard.schema.yml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_telemetry_postcard:
+    type: dict
+    keys:
+      disabled:
+        type: bool
+        description: Enable or disable the postcard telemetry feature
+      ingress:
+        type: dict
+        description: Configure ingress parameters
+        keys:
+          collection:
+            type: dict
+            description: Collector configuration
+            keys:
+              source:
+                type: str
+                description: GRE source configuration
+              destination:
+                type: str
+                description: GRE destination configuration
+              version:
+                type: int
+                description: Postcard version
+                valid_values: [ 1, 2 ]
+          sample:
+            type: dict
+            description: Configure sampling parameters
+            keys:
+              rate:
+                type: int
+                description: Configure sampling rate
+                convert_types:
+                - str
+                valid_values: [ 16384, 32768, 65536 ]
+              tcp_udp_checksum:
+                type: int
+                description: Configure TCP/UDP checksum or IP ID checksum
+                convert_types:
+                - str
+      profiles:
+        type: list
+        description: Configure the postcard telemetry profile
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Profile name
```

Comment:
Please add punctuation to all descriptions.

Reply:
Fixed